app/controllers/posts_controller.rb

```diff
@@ -33,6 +33,7 @@ def create
 
     respond_to do |format|
       if @post.save
+        @post.create_tags_from_tag_string
```

Comment:
I would move this to an `after_save` callback, so that you don't have to call this method from the controller: http://api.rubyonrails.org/classes/ActiveRecord/Callbacks.html